stdlib/statistics.pyi

```diff
@@ -94,7 +94,13 @@ else:
 def median(data: Iterable[_NumberT]) -> _NumberT: ...
 def median_low(data: Iterable[SupportsRichComparisonT]) -> SupportsRichComparisonT: ...
 def median_high(data: Iterable[SupportsRichComparisonT]) -> SupportsRichComparisonT: ...
-def median_grouped(data: Iterable[_NumberT], interval: _NumberT = ...) -> _NumberT: ...
+
+if sys.version_info >= (3, 12):
```

Comment:
Possibly 3.11 depending on https://github.com/python/cpython/pull/92533#issuecomment-1123056470

Reply:
Oh glad you asked about it, I wasn't sure so I just tried to say "is so far only on the 3.12 branch" loudly and see if Raymond commented